bbws/entity.py

```diff
@@ -33,19 +32,55 @@
 
 
 class EntityResource(Resource):
+    get_parser = reqparse.RequestParser()
+    get_parser.add_argument('revision', type=int, default=None)
+
+    entity_class = Entity
```

Comment:
EntityResource is, in this case, an "abstract" class. entity_class is set by the classes that inherit from it and used in the queries below. Setting it here was, I suppose, reflexive and not really necessary, since EntityResource shouldn't be used on its own.